src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    protected Object clone() throws CloneNotSupportedException {
+        return super.clone();
+    }
```

Comment:
Do we need `clone` method?

Reply:
It was auto generated. 
It isn't necessarily method. 
I deleted it.